source/tit/core/str.cpp

```
@@ -18,6 +22,46 @@ namespace tit {
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+auto str_is_identifier(std::string_view str) -> bool {
```

Cppcheck (2.20.0)

[style] The function 'str_is_identifier' is never used.

(unusedFunction)